concepts/05 Widgets/Form/01 Getting Started with Form/60 Modify the Form at Runtime.md

```diff
@@ -0,0 +1,178 @@
+You can change any form, editor, or item properties at runtime. The following code dynamically makes the entire Form read-only. In **jQuery**, use the [option(optionName, optionValue)](/Documentation/ApiReference/UI_Widgets/dxForm/Methods/#optionoptionName_optionValue) method to update the [readOnly](/Documentation/ApiReference/UI_Widgets/dxForm/Configuration/#readOnly) property's value. In **Angular**, **Vue** or **React**, bind the property's value to a component property.
```

Comment:
Go from the general to the specific. Say how to do something first, and only then say how it is done in this particular case. By the way, the `option` method changes only form properties. Item properties should be changed with the `itemOption` method.